ibis/expr/operations.py

```diff
@@ -2694,3 +2799,17 @@ def output_type(self):
 
     def root_tables(self):
         return distinct_roots(*self.values)
+
+
+# GEOMETRIC OPERATIONS
+
+class Distance(ValueOp):
+    """
+    Calculates distance in meters between two WGS-84 positions.
```

Comment:
What is WGS-84? Does it really need to be in the docstring here?

Reply:
> WGS84 is the standard U.S. Department of Defense definition of a global reference system for geospatial information and is the reference system for the Global Positioning System (GPS). 

I can remove if you want it is unnecessary

Reply:
What sort of constraints does this put on the operation? This is different than the standard distance calculation, right?

Reply:
Hm, I'm not comfortable adding a distance function that assumes what appears to be a domain specific coordinate system. Can we make this Euclidean distance instead and follow up after this PR about adding the WGS version?

Reply:
Yes, we can do that. We just cannot test it because mapd doesn't have a Euclidean operation. 
I am just not sure if common Euclidean distance function works with lat lon parameters .. so maybe would be better to remove this function and create a issue to follow this discussion.